Client/trunk/ParaEngineClient/BMaxModel/BMaxFrameNode.cpp

```diff
@@ -56,7 +56,7 @@ void ParaEngine::BMaxFrameNode::SetIndex(int nIndex)
 
 BMaxFrameNode* BMaxFrameNode::GetParent()
 {
-	if (m_nParentIndex >= 0)
+	if (m_nParentIndex >= 0 && m_pParser->m_nodes.find(m_nParentIndex) != m_pParser->m_nodes.end())
 	{
 		auto pParent = m_pParser->m_nodes[m_nParentIndex];
 		if (pParent)
```

Comment:
auto it = m_pParser->m_nodes.find(m_nParentIndex);
if (it != m_pParser->m_nodes.end())
{
 auto pParent = it->second;
}